download correct hostapd for architecture

georgejhunt opened the pull request:
I changed unleashkids to work with this for arm, i386, but not yet for x86_64

jvonau commented:
I understand the need with rpi2 but why not other arm proccessers and why
intel? Is this not in Fedora or is something broken? Trying to understand
the need for custom compiled software that needs long term support..

On Tue, Jul 21, 2015 at 1:49 PM, georgejhunt <notifications@github.com>
wrote:

> I changed unleashkids to work with this for arm, i386, but not yet for
> x86_64
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/XSCE/xsce/pull/357
> Commit Summary
>
>    - download correct hostapd for architecture
>
> File Changes
>
>    - *M* roles/network/tasks/hostapd.yml
>    <https://github.com/XSCE/xsce/pull/357/files#diff-0> (24)
>
> Patch Links:
>
>    - https://github.com/XSCE/xsce/pull/357.patch
>    - https://github.com/XSCE/xsce/pull/357.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/XSCE/xsce/pull/357>.
>

georgejhunt commented:
Good questions. I agree that we should only be substituting hostapd, when
the standard repo version does not meet our needs. In the two cases here,
I've had to compile a driver for the tplink wn725n, and the driver is not
compatible with stock hostapd.

And right now, that's only on rpi2 and xo1.5. I've changed the PR to what I
think is needed, but every time I look at it I see something I've missed.

On Tue, Jul 21, 2015 at 12:16 PM, Jerry Vonau <notifications@github.com>
wrote:

> I understand the need with rpi2 but why not other arm proccessers and why
> intel? Is this not in Fedora or is something broken? Trying to understand
> the need for custom compiled software that needs long term support..
>
> On Tue, Jul 21, 2015 at 1:49 PM, georgejhunt <notifications@github.com>
> wrote:
>
> > I changed unleashkids to work with this for arm, i386, but not yet for
> > x86_64
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/XSCE/xsce/pull/357
> > Commit Summary
> >
> > - download correct hostapd for architecture
> >
> > File Changes
> >
> > - *M* roles/network/tasks/hostapd.yml
> > <https://github.com/XSCE/xsce/pull/357/files#diff-0> (24)
> >
> > Patch Links:
> >
> > - https://github.com/XSCE/xsce/pull/357.patch
> > - https://github.com/XSCE/xsce/pull/357.diff
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/XSCE/xsce/pull/357>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/XSCE/xsce/pull/357#issuecomment-123450420>.
>
